package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "esn-frontend-account-inbox",
+    "version": "0.1.0",
+    "description": "Intermediary frontend module between Inbox and Account frontends for the OpenPaaS Suite - https://open-paas.org/",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build:dev": "webpack"
+    },
+    "author": "",
+    "license": "AGPLv3",
+    "dependencies": {
+        "angular": "1.3.20",
+        "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
+        "lodash": "2.4.1"
```

Comment:
I hope not

Reply:
wich version  should i use ?

Reply:
in theory esn-frontend-comon-libs comes with lodash and it install lodash in node_modules. I know that it does not work when esn-frontend-common-libs is npm-linked, but this is not a sufficient reason to add it there.

You can have a look at the other SPA, none is requiring  lodash

Reply:
Yes, @asmaNode please remove lodash dependency.